ios/TwilioVoiceReactNative.m

```diff
@@ -846,11 +848,13 @@ - (void)asyncPushRegistryInitialization:(dispatch_time_t)timeout
 #pragma mark - Bingings (Call Invite)
 
 RCT_EXPORT_METHOD(callInvite_accept:(NSString *)callInviteUuid
-                  acceptOptions:(NSDictionary *)acceptOptions
```

Comment:
did we change the signature in the RN layer as well?

Reply:
Yes, we changed that API when I was working on the Android layer. Here's the relevant line: https://github.com/twilio/twilio-voice-react-native/blob/task/opt-in-feature-ios/src/type/NativeModule.ts#L63

And the JS API (the public API) code is here: https://github.com/twilio/twilio-voice-react-native/blob/task/opt-in-feature-ios/api/voice-react-native-sdk.api.md?plain=1#L318 and here https://github.com/twilio/twilio-voice-react-native/blob/task/opt-in-feature-ios/api/voice-react-native-sdk.api.md?plain=1#L1022